packages/hd/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
```

Comment:
the changelog should be auto-generated, we don't need to edit it manually, we can rm it